core/core/domain.py

```diff
@@ -7,151 +7,152 @@
+    BIOPARK = "certificate:BIOPARK"
+    BIORE = "certificate:BIORE"
+    BK = "certificate:BIOKREIS"
+    BK_RF = "certificate:BIOKREIS_REGIONAL_FAIR"
+    BLUES_P = "certificate:BLUESIGN_PRODUCT"
+    BLUES_A = "certificate:BLUESIGN_APPROVED"
+    BONSUCRO = "certificate:BONSUCRO"
+    BSCI = "certificate:BUSINESS_SOCIAL_COMPLIANCE_INITIATIVE"
+    BVA = "certificate:BIOCYCLIC_VEGAN_AGRICULTURE"
+    CCCC = "certificate:COMMON_CODE_COFFEE_COMMUNITY"
+    CCS = "certificate:CONTENT_CLAIM_STANDARD"
+    CA_BC = "certificate:CA_BIOCOTTON"
+    CMIA = "certificate:COTTON_MADE_IN_AFRICA"
+    CNI = "certificate:CLIMATE_NEUTRAL_STANDARD"
+    CP = "certificate:CLIMATE_PARTNER"
+    CSE = "certificate:CERTIFIED_SUSTAINABLE_ECONOMICS"
+    CTC_T_BRONZE = "certificate:CRADLE_TO_CRADLE_BRONZE"
+    CTC_T_SILVER = "certificate:CRADLE_TO_CRADLE_SILVER"
+    CTC_T_GOLD = "certificate:CRADLE_TO_CRADLE_GOLD"
+    CTC_T_PLATIN = "certificate:CRADLE_TO_CRADLE_PLATINUM"
+    CTC_WR = "certificate:CRADLE_TO_CRADLE_DETERGENTS"
+    CTFL = "certificate:CARBON_TRUST_FOOTPRINT"
+    DEMETER = "certificate:DEMETER"
+    ECOCERT = "certificate:ECOCERT"
+    ECOVIN = "certificate:ECOVIN"
+    ECO_GARANTIE = "certificate:ECOGARANTIE"
+    ECO_INSTITUTE = "certificate:ECO_INSTITUTE"
+    ECO_PP_OEKO_TEX = "certificate:ECO_PASSPORT_OEKO_TEX"
+    ECO_VEG = "certificate:ECOVEG"
+    ENERGY_STAR = "certificate:ENERGY_STAR"
+    EPEAT_MP = "certificate:EPEAT_MOBILE_PHONES"
+    EPEAT_L_CO = "certificate:EPEAT_LAPTOPS"
+    EU_BIO = "certificate:EU_BIO"
+    EU_ECO = "certificate:EU_ECOLABEL"  # new
+    EU_ECO_H = "certificate:EU_ECOLABEL_HARD_SURFACES"
+    EU_ECO_L_CO = "certificate:EU_ECOLABEL_LAPTOPS"
+    EU_ECO_WR = "certificate:EU_ECOLABEL_DETERGENTS"
+    EU_ECO_P = "certificate:EU_ECOLABEL_PAPER"
+    EU_ECO_T = "certificate:EU_ECOLABEL_TEXTILES"
+    ETI = "certificate:ETHICAL_TRADING_INITIATIVE"
+    FC = "certificate:FAIRCHOICE"
+    FFL = "certificate:FAIR_FOR_LIFE"
+    FLA = "certificate:FAIR_LABOR_ASSOCIATION"
+    FNG = "certificate:FAIR_N_GREEN"
+    FR = "certificate:FLUSTIX_RECYCLED"
+    FS = "certificate:FAIR_STONE"
+    FSC = "certificate:FOREST_STEWARDSHIP_COUNCIL"
+    FSC_M = "certificate:FOREST_STEWARDSHIP_COUNCIL_MIX"
+    FSC_R = "certificate:FOREST_STEWARDSHIP_COUNCIL_RECYCLED"
+    FT = "certificate:FAIRTRADE"
+    FT_B = "certificate:FAIRTRADE_COTTON"
+    FT_CP = "certificate:FAIRTRADE_COTTON_PROGRAM"
+    FT_TP = "certificate:FAIRTRADE_TEXTILE_PRODUCTION"
+    FWF = "certificate:FAIR_WEAR_FOUNDATION"
+    G_IT = "certificate:FUJITSU_GREEN_IT"
+    GAEA = "certificate:GAEA_ECOLOGICAL_AGRICULTURE"
+    GCS = "certificate:GOOD_CASHMERE_STANDARD"
+    GEPA = "certificate:GEPA"
+    GGN = "certificate:GGN"
+    GK = "certificate:GREEN_BUTTON"
+    GOTS = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD"
+    GOTS_ORGANIC = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_ORGANIC"
+    GOTS_MWOM = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_ORGANIC"
```

Comment:
@se-jaeger I see. so how about having only 2 and both have the threshold in the ID?

```suggestion
    GOTS_ORGANIC = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_95"
    GOTS_MWOM = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_70"
```

Reply:
```suggestion
    GOTS = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD"
    GOTS_ORGANIC = "certificate:GLOBAL_ORGANIC_TEXTILE_STANDARD_ORGANIC"
```

this is another proposal. the first would be 70%+, second one 90%+. That's how I thought about it in the other file:
https://github.com/ecosia/core/blob/dfb3c5e1948f84dd5f5b6555a3c4ac7b6466c0fd/shopping/sustainability-labels.json#L1322-L1356

Reply:
Here: https://github.com/ecosia/core/blob/dfb3c5e1948f84dd5f5b6555a3c4ac7b6466c0fd/shopping/sustainability-labels.json#L1349
The ID and name do not fit.

Reply:
@se-jaeger I think I see what you mean. Does this fix it? https://github.com/ecosia/core/pull/9241